pandas/tests/indexes/datetimes/test_date_range.py

```diff
@@ -22,7 +22,10 @@
 
 def eq_gen_range(kwargs, expected):
     rng = generate_range(**kwargs)
-    assert (np.array_equal(list(rng), expected))
```

Comment:
I would remove this function and just directly compare these (only 3 uses)

Reply:
Done.